src/components/Icon/icons.ts

```diff
@@ -97,6 +99,7 @@ export const icons = {
   FilterLeft: BsFilterLeft,
   Giftcard: MdCardGiftcard,
   Globe: BiGlobe,
+  Google: AiFillGoogleCircle,
```

Comment:
This is not utilised yet that I can find in the code, though it _will_ be for the login with Google button, yes?

Reply:
yes this was the remnant of the custom UI form - would delete this